Add missing sections to release notes (#529)
kyma-project/website: content/blog-posts/2020-01-09-release-notes-1.9/index.md

- [Compass](#compass) - Provisioner enhanced, Application Templates added, redesigned list view in the Compass UI, and more
- [Console](#console) - Improved Addons Configuration view
- [Eventing](#eventing) - Knative Eventing Mesh available for testing purposes

> - [Installation & Documentation](#installation-documentation) - Gardener provisioning based on Kyma CLI documented, a finite number of revisions set in Tiller

Reviewer: ```suggestion
- [Installation & Documentation](#installation-documentation) - Gardener provisioning based on Kyma CLI documented, a finite number of revisions set in Tiller
```
do we use articles here? 🤔

Author: true, I will remove it

After introducing Gardener cluster provisioning through the Kyma CLI in the previous release, in this release, we added the official documentation for this feature. The instructions for provisioning a Gardener cluster on GCP or Azure are now based on Kyma CLI. Check it out in [this](https://kyma-project.io/docs/1.9/root/kyma/#installation-install-kyma-on-a-cluster-prepare-the-cluster) document under the **Gardener** tab.

### Tiller now keeps a finite number of revisions

> Up until now, Tiller kept an infinite number of revisions. This resulted at times in Helm not being able to upgrade charts after many retries. Setting a limit to the number of revisions, resolved these issues. Another advantages of keeping only a few revisions is lower memory consumption for controllers listening on ConfigMap resources and simply fewer resources for Kubernetes to manage.

Reviewer: ```suggestion
Up until now, Tiller kept an infinite number of revisions. This resulted at times in Helm not being able to upgrade charts after many retries. Setting a limit to the number of revisions, resolved these issues. Another advantage of keeping only a few revisions is lower memory consumption for controllers listening on ConfigMap resources and fewer resources for Kubernetes to manage.
```

- Duplicate all old resources in the new component, under the new name and API groups,
- Remove the Asset Store and Headless CMS components after successful migration.

To guarantee the necessary data duplication and fulfil the need to run all components during the upgrade, make sure your cluster has enough resources. The migration scripts are available [here](https://github.com/kyma-project/kyma/tree/release-1.9/resources/rafter/templates).

>**NOTE:** Old CustomResourceDefinitions (CRDs) are not removed automatically during the upgrade process. Leaving the CRDs does not influence the upgrade. These CRDs will be removed with 1.10 release. However, you can remove them manually anytime using this command:
>
>```bash
>kubectl delete crd clusterdocstopics.cms.kyma-project.io && kubectl delete crd docstopics.cms.kyma-project.io && kubectl delete crd clusterbuckets.assetstore.kyma-project.io && kubectl delete crd buckets.assetstore.kyma-project.io && kubectl delete crd clusterassets.assetstore.kyma-project.io && kubectl delete crd assets.assetstore.kyma-project.io
>```

### Tiller

> As we modified the Tiller Deployment by adding a new environment variable that sets the Tiller revisions limit, you should apply the new Tiller resource before performing the upgrade:

Reviewer: ```suggestion
As we modified the Tiller Deployment by adding a new environment variable that sets the Tiller revisions limit, apply the new Tiller resource before performing the upgrade:
```

```bash

> kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/$KYMA_VERSION/installation/resources/tiller.yaml

Reviewer 2: ```suggestion
kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/1.9.0/installation/resources/tiller.yaml
```
